Content.Client/UserInterface/Systems/Ghost/GhostUIController.cs

```diff
@@ -21,7 +19,9 @@ public sealed class GhostUIController : UIController, IOnSystemChanged<GhostSyst
     [Dependency] private readonly IConfigurationManager _cfg = default!; // Frontier
 
     [UISystemDependency] private readonly GhostSystem? _system = default;
-    [UISystemDependency] private readonly RespawnSystem? _respawn = default; // Frontier
+
+    // Updated when get death time from the server.
```

Comment:
needs to indicate where it's from!
```suggestion
    // DeltaV: Updated when get death time from the server.
```